packages/frontend/src/components/Game/GameBoard/TileGrid/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { mockPhotos } from 'components/__mocks__/mockPhotos';
+import './style.scss';
+import Tile from '../Tile';
+
+const TileGrid = () => {
+  const photosCopy = [...mockPhotos];
+  const tiles = mockPhotos.concat(photosCopy);
+  console.log('potoscopy', tiles);
+
+  console.log(tiles[0].src.small, 'after');
+
+  const shuffleTiles = () => {
+    let currentIndex = tiles.length,
+      randomIndex;
+
+    while (currentIndex > 0) {
+      randomIndex = Math.floor(Math.random() * currentIndex);
+      currentIndex--;
+
+      [tiles[currentIndex], tiles[randomIndex]] = [
+        tiles[randomIndex],
+        tiles[currentIndex],
+      ];
+    }
+    return tiles;
+  };
+
+  shuffleTiles(tiles);
+  return (
+    <div id="grid" className="tile_grid d-flex align-items-center flex-wrap">
+      {tiles.map((tile) => (
+        <Tile
+          id={tile.id}
+          key={Math.random()}
```

Comment:
This key could just be the `index` for the loop. `tiles.map((tile, index) => ...)`

Reply:
This all looks good, are there any tests written for the TIle or TileGrid components?